website/src/index.css

```diff
@@ -210,6 +210,12 @@ footer li:last-child::after {
   grid-template-columns: 1fr;
 }
 
+.sponsor-image {
```

Comment:
wouldn't an `align-self` and `justify-self` on the `.sponsor img` be simpler? Would also remove the need for the div.

Reply:
You're right! Changes pushed